app/commands/mentor/update_stats.rb

```diff
@@ -21,10 +19,18 @@ def call
     ]
     new_values = old_values.clone
 
-    new_values[0] = User::ResetCache.(mentor, :num_students_mentored) if update_counts
-    new_values[1] = User::ResetCache.(mentor, :num_solutions_mentored) if update_counts
-    new_values[2] = Mentor::UpdateSatisfactionPercentage.(mentor) if update_satisfaction_rating
+    new_values[0] = User::ResetCache.(mentor, :num_students_mentored)
+    new_values[1] = User::ResetCache.(mentor, :num_solutions_mentored)
+    new_values[2] = Mentor::UpdateSatisfactionPercentage.(mentor)
 
-    User::UpdateMentorRoles.defer(mentor) if old_values != new_values
```

Comment:
This command got renamed to `UpdateSupermentorRole`, which is still called here, but just below.